docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -174,31 +174,54 @@ initialize the model.
      - Measure
      - Value
      - Notes
-   * - State
+   * - S
      - prevalence
+     - 1 - prevalence_c426
      - 
-     - 
-   * - State
+   * - S
      - birth prevalence
+     - 0
      - 
-     - 
-   * - State
+   * - S
      - excess mortality rate
+     - 0
      - 
-     - 
-   * - State
+   * - S
      - disabilty weights
-     - 
+     - 0
      -
-   * - ALL
-     - cause specific mortality rate
+   * - I
+     - prevalence
+     - prevalence_c426
      - 
+   * - I
+     - birth prevalence
+     - 0
      - 
-
-Transition Data
-"""""""""""""""
-
-This table contains the measures needed for transition from one state to other in the cause model. 
+   * - I
+     - excess mortality rate
+     - :math:`\text{csmr_c426}\times\text{prevalence_c426}`
+     - 
+   * - I
+     - disabilty weights
+     - :math:`\displaystyle{\sum_{s\in\text{s_c426}}}\scriptstyle{\text{disability_weight}_s\,\times\,\text{prev}_s}`
```

Comment:
I think what software engineer needs is a weighted average across all sequelae within c426, we might want to add SUM(prev_s) as denominator.